charms/ambassador-auth/reactive/ambassador_auth.py

```diff
@@ -0,0 +1,97 @@
+import hashlib
+import random
+import string
+from base64 import b64encode
+
+from subprocess import run
+
+run(['apt', 'update'], check=True)
+run(['apt', 'install', 'python3-cffi-backend'], check=True)
```

Comment:
@johnsca is there a better way of handling this? The operator pod needs some apt packages installed for `bcrypt`.